building/trusted_applications.rst

```diff
@@ -373,35 +373,50 @@ third step the binary and its signature are stitched together into the full TA.
 Offline Signing of TAs
 ======================
 
-The TA dev kit does sign an application as last step of the linking process. For
-example, the file ``ta/arch/arm/link.mk`` in the :ref:`optee_os` source tree
-contains the statement
+There are two types of TAs that can be signed offline: the in-tree TAs, which come
+with the OP-TEE OS (for example the ``pkcs11`` TA) and any external TA coming from the user. Offline signing
+runs directly on `.stripped.elf` TA files and there is a slight difference as how they are
+generated. The in-tree `.stripped.elf` TA files are generated during the OP-TEE OS compilation, while
+with external TAs it is usually done by the user. In both cases however, the signing process is the
```

Comment:
while with ... the same. -> Compilation of other TAs is done as explained above. The offline signing process does not differ between internal and regular TAs.

Reply:
See [here](https://github.com/OP-TEE/optee_docs/pull/125#discussion_r638798844).